module/data/actor/templates/common.mjs

```diff
@@ -149,6 +149,7 @@ export default class CommonTemplate extends ActorDataModel.mixin(CurrencyTemplat
 
       abl.save = abl.mod + abl.saveBonus;
       if ( Number.isNumeric(abl.saveProf.term) ) abl.save += abl.saveProf.flat;
+      abl.attack = abl.mod + prof;
```

Comment:
If this is meant only for display purposes, should it not rather be a string? A numeric value won't support proficiency dice.

Reply:
It's a number on `ClassData#spellcasting` also, so we should be consistent. And speaking of consistency, we have `ClassData#spellcasting#save` rather than `dc`. Not sure which one we should align on, but I think I'd lean towards `dc` personally.